tests/cnf/core/network/metallb/tests/metallb-crds.go

```diff
@@ -98,7 +101,7 @@ var _ = Describe("MetalLb New CRDs", Ordered, Label("newcrds"), ContinueOnFailur
 		bfdProfile := createBFDProfileAndVerifyIfItsReady(frrk8sPods)
 
 		createBGPPeerAndVerifyIfItsReady(tsparams.BgpPeerName1, ipv4metalLBIPList[0], bfdProfile.Definition.Name,
-			tsparams.LocalBGPASN, false, 0, frrk8sPods)
+			tsparams.RemoteBGPASN, false, 0, frrk8sPods)
```

Comment:
Same here

Reply:
It was a workaround for the BGP+BFD bug, that was fixed recently in 4.19. Reverted it back